app/controllers/prison/visits_controller.rb

```diff
@@ -72,4 +80,30 @@ def visit_page(visit)
       prison_deprecated_visit_path(visit)
     end
   end
+
+  def load_visit
```

Comment:
Same pattern as above. Memoizing the scope an renaming/aliasing it to visit. It would also read a lot better line 51 and 8